### ISSUE TITLE
The value of the KL Loss

### ISSUE THREAD

**Issue by pangsu0613** [yihui-he/KL-Loss#27]
Hello Yihui, Thank you very much for sharing the code, it is an excellent work! I have a question about the value of the KL loss. I did read the instructions before creating this new issue, but I think my question is a little bit different. Here is my question:
KL divergence is zero only when the two distributions are identical. But as we know, on the one hand, our predictions can never be identical to the ground truth, on the other hand, we discard the terms (H(P_D(x)) and log(2pi)/2) that are not depended on the estimated parameters. This would make the KL loss not converge to zero (or any fixed value), ideally, it would converge to a minimum, but we don't know the value of this minimum. So, my question is how you determine if the training goes well (since unlike other types of loss function which would converge to a fixed value, like 0, we don't know the ideal value of the proposed KL loss)? and how to evaluate the variance (since we don't have the ground truth for variance)? Thank you.

**Comment by ethanhe42** [comment]
- We can inspect the loss curve. In addition, you can still calculate the standard loss as a metric to see if training goes well.
- We can only visually verify since we don't have the GT. Ideally, ask multiple labelers to label a bounding box and calculate the ground truth variance.

**Comment by pangsu0613** [comment]
Thank you for your quick response.
